### DISCUSSION TITLE
Keep getting error type: madmax

### DISCUSSION THREAD

**Discussion by Tongie969** [ericaltendorf/plotman#876] [Q&A]
Hi guys,

after some, a while plotting without plotman I would love to use it again. 

Of course, I would love to use madmax with it  but I keep getting this output. 

![image](https://user-images.githubusercontent.com/63996344/126048097-f450a08a-eb93-4244-a7cb-4cf2474b4b40.png)

I am not sure if config.yamal is fine 

![image](https://user-images.githubusercontent.com/63996344/126048122-8f74557c-b328-46e3-a6f9-8078bf66ece8.png)

Does this make sense? 

Thank you very much in advance and happy plotting

**Comment by altendky** [comment]
You need consistent indentation.  Your `farmer_pk:` and `pool_contract_address:` lines are indented a space too far compared to the rest.

**Comment by Tongie969** [comment]
Thanks for your fast response @altendky the yamal validator said it's fine but not for placement. I managed to produce another error.

**Reply by altendky** [reply]
Would you like help with the other error?

**Reply by Tongie969** [reply]
> Would you like help with the other error?

That would be such great :) 

![image](https://user-images.githubusercontent.com/63996344/126081275-7be6ac49-957a-4cda-aa29-4d4404689704.png)

**Reply by altendky** [reply]
`\t` refers to a tab.  Use consistent indentation.  I use spaces.  (Maybe YAML requires spaces not tabs?  I'm not sure...)

**Reply by Spaikys** [reply]
i had this problem in past, do not use TAB only space to avoid this error.

**Reply by Tongie969** [reply]
Thank you very much guys. 

I was planning to setup plotman with madmax for archiving before my holiday. Now I just have remote access and only one drive left to fill. 

But I will go over the config when I am back home. 

Thanks for your help :)
